content/id/docs/contribute/participate/_index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Berpartisipasi dalam SIG Docs
+content_type: concept
+weight: 60
+card:
+  name: contribute
+  weight: 60
+---
+
+<!-- overview -->
+
+SIG Docs merupakan salah satu
+[kelompok peminatan khusus (_special interest groups_)](https://github.com/kubernetes/community/blob/master/sig-list.md)
+dalam proyek Kubernetes, yang berfokus pada penulisan, pembaruan, dan pemeliharaan
+dokumentasi untuk Kubernetes secara keseluruhan. Lihatlah
+[SIG Docs dari repository github komunitas](https://github.com/kubernetes/community/tree/master/sig-docs)
+untuk informasi lebih lanjut tentang SIG.
+
+SIG Docs menerima konten dan ulasan dari semua kontributor. Siapa pun dapat membuka
+_pull request_ (PR), dan siapa pun boleh mengajukan masalah (_issue_) tentang konten atau komentar
```

Comment:
nit: I think masalah is always "negative", but github "issue" can also contain positive things, usually we translate to [isu](https://kbbi.web.id/isu0
```suggestion
_pull request_ (PR), dan siapa pun boleh mengajukan isu tentang konten atau komentar
```

Reply:
Ok! nice wording "isu"